src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/sections/orthologues/Orthologues.tsx

```diff
@@ -0,0 +1,235 @@
+  // const showBestMatches = useCallback(() => {
+  //   props.setOrthologueShowBestMatches(!props.shouldShowBestMatches);
+
+  //   props.fetchOrthologueSpecies(
+  //     props.orthologueSearchTerm,
+  //     !props.shouldShowBestMatches,
+  //     props.shouldShowAll,
+  //     props.orthologueSpecies
+  //   );
+
+  // }, [props.shouldShowBestMatches, props.orthologueSpecies]);
+
+  // const showAll = useCallback(() => {
+  //   props.setOrthologueShowAll(!props.shouldShowAll);
+  //   props.fetchOrthologueSpecies(
+  //     props.orthologueSearchTerm,
+  //     props.shouldShowBestMatches,
+  //     !props.shouldShowAll,
+  //     props.orthologueSpecies
+  //   );
+
+  // }, [props.shouldShowAll, props.orthologueSpecies]);
+
+  const getResultCounter = () => {
+    const totalSpecies = getAttributesCount(props.orthologueSpecies);
+    return (
+      <>
+        <span>{totalSpecies ? totalSpecies : 0}</span>
+        <span> result{totalSpecies > 1 ? 's' : ''}</span>
+      </>
+    );
+  };
+
+  return (
+    <div className={styles.orthologues}>
+      <div className={styles.filterWrapper}>
+        <label>Find</label>
+        <Input
+          value={props.orthologueSearchTerm}
+          onChange={inputOnChangeHandler}
+          placeholder={'Species'}
+        />
+        {props.orthologueSearchTerm.length > 0 && (
+          <span>
+            <span className={styles.resultCounter}>{getResultCounter()}</span>
+            {/* <span
+              className={styles.bestMatchesFilter}
+              onClick={showBestMatches}
+            >
+              {props.shouldShowBestMatches
+                ? 'View all'
+                : 'View best matches'}
+            </span> */}
+            {/* <span className={styles.showAllFilter} onClick={showAll}>
+              {props.shouldShowAll ? 'Hide all' : 'Show all'}
+            </span> */}
+          </span>
+        )}
+      </div>
+
+      {!!props.orthologueSpecies && (
+        <div>
+          <CheckBoxGrid
+            checkboxOnChange={speciesOnChangeHandler}
+            gridData={props.orthologueSpecies}
+            hideTitles={props.hideTitles}
+            columns={3}
+          />
+        </div>
+      )}
+
+      {!!props.orthologueAttributes &&
+        Object.keys(props.orthologueAttributes).map(
+          (species: string, key: number) => {
+            return (
+              <div key={key}>
+                <div className={styles.speciesAttributesSectionTitle}>
+                  <span>{species}</span>
+                </div>
+                <CheckBoxGrid
+                  checkboxOnChange={attributesOnChangeHandler}
+                  gridData={{ [species]: props.orthologueAttributes[species] }}
+                  hideTitles={true}
+                  columns={3}
+                />
+              </div>
+            );
+          }
+        )}
+    </div>
+  );
+};
+
+type DispatchProps = {
+  setOrthologueAttributes: (setOrthologueAttributes: {}) => void;
```

Comment:
what's the type of setOrthologueAttributes?

Reply:
That's now updated